docs/hot_module_reloading.md

```diff
@@ -1,3 +1,5 @@
+# NOTICE: THESE HOOKS WILL LIKELY SOON BE DEPRECATED/DELETED 
```

Comment:
I might drop the "likely"

We could also add a note below with something like "if this breaks your use case please file an issue at ..."

Reply:
done